docs/guides/mozilla-sops.md

```diff
@@ -95,15 +86,91 @@ Create a kustomization for reconciling the secrets on the cluster:
 ```sh
 flux create kustomization my-secrets \
 --source=my-secrets \
+--path=./clusters/cluster0 \
 --prune=true \
 --interval=10m \
 --decryption-provider=sops \
 --decryption-secret=sops-gpg
 ```
 
-Note that the `sops-gpg` can contain more than one key, sops will try to decrypt the
+Note that the `sops-gpg` can contain more than one key, SOPS will try to decrypt the
 secrets by iterating over all the private keys until it finds one that works.
 
+## Optional: Export the public key into the Git directory
+
+Commit the public key to the repository so that team members who clone the repo can encrypt new files:
+
+```sh
+gpg --export --armor "${KEY_FP}" > ./clusters/cluster0/.sops.pub.asc
+```
+
+Check the file contents to ensure it's the public key before adding it to the repo and committing.
+
+```sh
+git add ./clusters/cluster0/.sops.pub.asc
+git commit -am 'Share GPG public key for secrets generation'
+```
+
+Team members can then import this key when they pull the Git repository:
+
+```sh
+gpg --import ./clusters/cluster0/.sops.pub.asc
+```
+
+!!! hint
+    The public key is sufficient for creating brand new files.
+    The secret key is required for decrypting and editing existing files because SOPS computes a MAC on all values.
+    When using solely the public key to add or remove a field, the whole file should be deleted and recreated.
+
+## Configure the Git directory for encryption
+
+Write a [SOPS config file](https://github.com/mozilla/sops#using-sops-yaml-conf-to-select-kms-pgp-for-new-files) to the specific cluster or namespace directory used
+to store encrypted objects with this particular GPG key's fingerprint.
+
+```yaml
+cat <<EOF > ./clusters/cluster0/.sops.yaml
```

Comment:
This breaks the reconciliation, users have to add `.sops.yaml` to `.sourceingnore`.

Reply:
Good catch -- I didn't notice this because I put `.sops.yaml` in my `cluster0/flux-system/` directory where it gets ignored by the `kustomization.yaml`.

I'm thinking that's not a good sole recommendation even if it steps around the problem.
We can add a step here that adds `**/.sops.yaml` to the `.sourceignore`.

Have we considered before, adding a default `.sourceignore` to bootstrap repos?

Reply:
Prompting the user to do this here:
https://github.com/stealthybox/flux2/blob/sops-gpg-batch/docs/guides/mozilla-sops.md#L151-L158